core/storage/recommendations/gae_models.py

```diff
@@ -37,6 +37,26 @@ class ExplorationRecommendationsModel(
     recommended_exploration_ids = ndb.StringProperty(
         repeated=True, indexed=False)
 
+    @staticmethod
+    def get_deletion_policy():
+        """Exploration recommendations are deleted only if the corresponding
+        exploration is not public.
+        """
+        return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
+
+    @staticmethod
+    def has_reference_to_user_id(unused_user_id):
+        """ExplorationRecommendationsModel doesn't reference any user_id
+        directly.
+
+        Args:
+            unused_user_id: str. The models doesn't reference user at all.
```

Comment:
Here and elsewhere, the description should still be "The (unused) ID of the user whose data should be checked."